packages/fxa-settings/.storybook/decorators.tsx

```diff
@@ -0,0 +1,18 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { DecoratorFn } from '@storybook/react';
+import AppLocalizationProvider from 'fxa-react/lib/AppLocalizationProvider';
+
+// Receive a component and apply a bordered container with max width of 400px.
```

Comment:
Is this comment intentional? Maybe it should say something about applying ftl?

Reply:
Not intentional at all... I'll replace this.